elastica/joint.py

```diff
@@ -209,9 +210,14 @@ class FixedJoint(FreeJoint):
             Damping coefficient of the joint.
         kt: float
             Rotational stiffness coefficient of the joint.
+        nut: float
+            Rotational damping coefficient of the joint.
+        static_rotation: np.array
```

Comment:
I feel `static_rotation` is less intuitive and maybe `rest_rotation_matrix` is more intuitive? since it is the rest orientation of the 2 connected elements.

Reply:
Resolved in https://github.com/GazzolaLab/PyElastica/pull/135/commits/9a5cf1cb90ebd7fb6a215d799eb27c5d0bd1c5f0